src/SFML/Window/Window.cpp

```diff
@@ -55,13 +55,21 @@ Window::Window(WindowHandle handle, const ContextSettings& settings)
 }
 
 
+////////////////////////////////////////////////////////////
+sf::Window::Window(Window&&) noexcept = default;
+
+
 ////////////////////////////////////////////////////////////
 Window::~Window()
 {
     close();
```

Comment:
Typically when there is a non-default destructor we need non-default move operations. How are you sure the default operations are the correct choice?

Reply:
yeah, it is problematic because of the fullscreen and future changes on close function. I will take a look and fix it.

Reply:
I think we need to make sure that if the window is fullscreen that the address of the new window is used as the address of the current fullscreen window instead of the moved-from window. Other than that I think we can rely on the default move semantics for the remaining data members of `sf::WindowBase` and `sf::Window`.

Reply:
I did the fullscreen check on `sf::WindowBase`  move constructor and call close in both move after that for future safety.

Reply:
I'd rather not preemptively add code we don't know if we'll ever need.